test/unit-tests.js

```diff
@@ -62,6 +62,14 @@ describe('midi.input', function() {
         input.openPort(999);
       }).should.throw('Invalid MIDI port number');
     });
+
+    it('should return true if not already open', function() {
+      input.openPort(0).should.be.Boolean.and.eql(true);
```

Comment:
Is there a reason you're checking the type is Boolean? Any reason not to just check for `.eql(true)` or `.eql(false)`?